src/Features/UserImpersonation.php

```diff
@@ -48,6 +48,23 @@ public static function makeResponse(string|ImpersonationToken $token, int $ttl =
 
         $token->delete();
 
+        session()->put('tenancy_impersonation', true);
+
         return redirect($token->redirect_url);
     }
+
+    public static function isImpersonating(): bool
+    {
+        return session()->has('tenancy_impersonation');
+    }
+
+    /**
+     * Logout and forget session.
```

Comment:
```suggestion
     * Logout from the tenant domain and forget session.
```
Would this be more accurate?

Reply:
> Would this be more accurate?

It can't be domain always. This action can be performed when using other identification methods.

Reply:
Will just go with this then, per our BC convo:
```suggestion
     * Logout from the current domain and forget impersonation session.
```